packages/@aws-cdk/cicd/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@aws-cdk/cicd",
+  "description": "Continuous Integration / Continuous Delivery for CDK Applications",
+  "version": "0.14.0",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "targets": {
+      "java": {
+        "maven": {
+          "groupId": "software.amazon.awscdk",
+          "artifactId": "cdk-cicd"
+        },
+        "package": "software.amazon.awscdk.cicd"
+      },
+      "sphinx": {},
+      "dotnet": {
+        "namespace": "Amazon.CDK.CICD",
+        "signAssembly": true,
+        "assemblyOriginatorKeyFile": "../../key.snk"
+      }
+    },
+    "outdir": "dist"
+  },
+  "scripts": {
+    "build": "cdk-build",
+    "package": "cdk-package",
+    "pkglint": "pkglint -f",
+    "test": "cdk-test",
+    "watch": "cdk-watch",
+    "integ": "cdk-integ"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-cloudformation": "^0.14.0",
+    "@aws-cdk/aws-codepipeline-api": "^0.14.0",
+    "@aws-cdk/cdk": "^0.14.0",
+    "@aws-cdk/cx-api": "^0.14.0"
+  },
+  "devDependencies": {
```

Comment:
The ordering of the top-level properties here is not consistent with the other packages (`dependencies` before `devDependencies`, `author`, `homepage`, etc. at the end, ...). No biggie though.

Reply:
Yeah well I actually prefer this order (interesting stuff first) to the average we have everywhere... But it's true we probably should standardize at some point...